meinberlin/assets/scss/components_user_facing/_control-bar.scss

```diff
@@ -23,12 +23,4 @@
     .button {
         margin-left: 0;
     }
-
-    @media screen and (max-width: var(--breakpoint-tablet)) {
```

Comment:
`var()` doesn't work in `@media` queries as far as I know? Adding the real breakpoint seems to distort the buttons so I think we should remove this for now

Reply:
I actually added the BO breakpoints (e.g. `breakpoint-tablet`) but then ended up removing them again as I didn't really need them and was wondering if there was a good reason why they haven't been added yet. But I agree we should add them then rather using actual numbers as it's much harder to read